cishouseholds/pipeline/survey_responses_version_digital_ETL.py

```diff
@@ -12,21 +12,45 @@ def digital_specific_transformations(df):
     df = df.withColumn("visit_datetime", F.lit(None).cast("timestamp"))  # Placeholder for 2199
 
     df = transform_survey_responses_generic(df)
-    df = clean_barcode_simple(df, "swab_sample_barcode_user_entered")
-    df = clean_barcode_simple(df, "blood_sample_barcode_user_entered")
-
-    df = df.withColumn("times_outside_shopping_or_socialising_last_7_days", F.lit(None))
-    df = edit_to_sum_or_max_value(
-        df=df,
-        column_name_to_assign="times_outside_shopping_or_socialising_last_7_days",
-        columns_to_sum=[
-            "times_shopping_last_7_days",
-            "times_socialising_last_7_days",
-        ],
-        max_value=7,
-    )
-    df = df.withColumn(
-        "work_not_from_home_days_per_week",
-        F.greatest("work_not_from_home_days_per_week", "education_in_person_days_per_week"),
+    dont_know_columns = [
+        "work_in_additional_paid_employment",
+        "work_nursing_or_residential_care_home",
+        "work_direct_contact_patients_or_clients",
+        "self_isolating",
+        "illness_lasting_over_12_months",
+        "ever_smoked_regularly",
+        "currently_smokes_or_vapes",
+        "cis_covid_vaccine_type_1",
+        "cis_covid_vaccine_type_2",
+        "cis_covid_vaccine_type_3",
+        "cis_covid_vaccine_type_4",
+        "cis_covid_vaccine_type_5",
+        "cis_covid_vaccine_type_6",
+        "other_household_member_hospital_last_28_days",
+        "other_household_member_care_home_last_28_days",
+        "hours_a_day_with_someone_else_at_home",
+        "physical_contact_under_18_years",
+        "physical_contact_18_to_69_years",
+        "physical_contact_over_70_years",
+        "social_distance_contact_under_18_years",
+        "social_distance_contact_18_to_69_years",
+        "social_distance_contact_over_70_years",
+        "times_hour_or_longer_another_home_last_7_days",
+        "times_hour_or_longer_another_person_your_home_last_7_days",
+        "times_shopping_last_7_days",
+        # "times_socialising_last_7_days",
+        "face_covering_work_or_education",
+        "face_covering_other_enclosed_places",
+        # "cis_covid_vacine_type",
+    ]
+    df = assign_raw_copies(df, dont_know_columns)
+    mapping_dict = {
+        "Prefer not to say": None,
+        "Don't Know": None,
```

Comment:
From the updated spec, I think this should include the response "I don't know the type"